semantic-python/semantic-python.cabal

```diff
@@ -23,6 +23,8 @@ common haskell
   build-depends:       base ^>=4.12
                      , fused-effects ^>= 0.5
                      , semantic-core ^>= 0.0
+                     , pathtype ^>= 0.8.1
```

Comment:
I found myself threading FilePath parameters all hither and yon, and before long, I found myself confused as heck. A typed path library alleviated this. I picked `pathtype` because it’s maintained, it contains helpers for directory I/O, it doesn’t require any TH splices, and it’s pretty comprehensible.